tests/framework/helpers/ArrayHelperTest.php

```diff
@@ -810,20 +810,22 @@ public function testGetValueObjects()
 
     /**
      * This is expected to result in a PHP error.
-     * @expectedException \PHPUnit_Framework_Error
+     * @requires PHPUnit 6.0
      */
     public function testGetValueNonexistingProperties1()
     {
+        $this->expectException('PHPUnit\Framework\Error\Notice');
```

Comment:
We could use `class_exists()` to detect expected exception and make this test work on phpunit <6.0

Reply:
I think it's not so important and could be skipped